form/unit_testing.rst

```diff
@@ -185,10 +185,10 @@ allows you to return a list of extensions to register::
 
         protected function getExtensions()
         {
-            $this->validator = $this->createMock(ValidatorInterface::class);
+            $validator = $this->createMock(ValidatorInterface::class);
             // use getMock() on PHPUnit 5.3 or below
-            // $this->validator = $this->getMock(ValidatorInterface::class);
-            $this->validator
+            // $validator = $this->getMock(ValidatorInterface::class);
+            $validator
                 ->method('validate')
                 ->will($this->returnValue(new ConstraintViolationList()));
             $validator
```

Comment:
It might be better to change the other spots back to how they were before, and fix this line to be `$this->validator`. They are definitely inconsistent (and should be fixed), but there is some value on setting it on a property (your tests can further configure the `$this->validator` mock.